packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -325,6 +337,18 @@ class MediaQueryData {
   /// Whether the user requested a high contrast between foreground and background
   /// content on iOS, via Settings -> Accessibility -> Increase Contrast.
   ///
+  /// Setting it, for instace by using: 
```

Comment:
Same here. Rather than duplicating these, you can create a `@template` and re-use it. :)

Reply:
Thanks for the comments. I tried `@template` but I could not get it to work as I needed, the code is not the same for the three examples.